### DISCUSSION TITLE
SMP support for Armv8-R AArch32 FVP

### DISCUSSION THREAD

**Discussion by ayankuma** [zephyrproject-rtos/zephyr#46385] [Q&A] [answered]
Hi,

With reference to https://github.com/zephyrproject-rtos/zephyr/pull/43581/commits which adds support for Armv8-R AArch32 FVP. Can you confirm if SMP is supported or not ?

By looking at https://github.com/iotbzh/zephyr/blob/6e4f6b72b7906235251ee54bf9cfcaae9b58d021/arch/arm/core/aarch32/cortex_a_r/reset.S , it seems SMP is not supported as there is no code to boot up secondary cpus. However, the dts has four cpu nodes. 

Can someone please confirm ?

**Comment by carlocaione** [comment, marked as answer]
Not supported yet.
